ui/src/components/admin/AdminForm.js

```diff
@@ -1,19 +1,9 @@
-import { Flex, Heading, useColorModeValue } from "@chakra-ui/react";
+import FormManager from "../forms/FormManager";
 
 export default function AdminForm(props) {
   return (
     <>
-      <Flex
-        height="85vh"
-        bg={useColorModeValue("gray.200")}
-        justifyContent="center"
-      >
-        <Flex direction="column" p={12} rounded={6}>
-          <Heading mb={6} alignItems="center">
-            Forms
-          </Heading>
-        </Flex>
-      </Flex>
+      <FormManager />
```

Comment:
hmm, code wise is fine. but don't particularly like the fact that when going to /admin/forms without logging in, it shows a 404 instead of redirect to login form